thunder/examine/__init__.py

```diff
@@ -216,3 +216,48 @@ def get_fusion_symbols(trace: TraceCtx, warn_if_fusions_unavailable: bool = True
             fusions.append(bsym)
 
     return fusions
+
+
+def get_nvFuser_repro(trace: TraceCtx, fusion_name: str, /) -> str:
```

Comment:
I think we should get the fusions out of obscurity (I think even `trace.get_trace_object(name)` to get `trace.python_ctx()[name]` would do the trick) and then make this a method of the trace.
This would enhance discoverability. WDYT? (also cc @lantiga who mentioned that he wanted an easier way on our podcast)

Also Nit: let's keep lowercase function names, please. I know every exception is worthy in itself, but as a user of APIs, having exceptions at all is terrible.

Reply:
Yeah I agree it's better to have a clear path to get to the fusion information from the trace without jumping hoops